src/features/auth/pages/login/Login.web.test.tsx

```diff
@@ -36,26 +36,30 @@ describe('<Login/>', () => {
     })
   })
 
-  it('should display forced login help message when the query param is given', () => {
+  it('should display forced login help message when the query param is given', async () => {
     useRoute.mockReturnValueOnce({ params: { displayForcedLoginHelpMessage: true } })
-    const { queryByRole } = renderLogin()
+    renderLogin()
 
-    const snackBar = queryByRole('status')
+    const snackBar = screen.queryByRole('status')
```

Comment:
Tu peux utiliser `findBy` ici, pour enlever le `waitFor` qui suit
```suggestion
    const snackBar = await screen.findByRole('status')
```